packages/field-base/src/label-for-controller.js

```diff
@@ -5,10 +5,10 @@
  */
 
 /**
- * A controller to link an input element with a slotted `<label>` element.
+ * A controller to link a label element with a slotted `<input>` element.
  */
-export class AriaLabelController {
-  constructor(host, input, label) {
+export class LabelForController {
```

Comment:
I would pick a more generic name, because `for` is rather an implementation detail.
Some inspiration from WHATWG HTML Standard - [§ 4.10.4 The `label` element](https://html.spec.whatwg.org/multipage/forms.html#the-label-element).

> The label element represents a caption in a user interface. The caption can be associated with a specific form control, known as the label element's labeled control, either using the for attribute, or by putting the form control inside the label element itself.

> Except where otherwise specified by the following rules, a label element has no labeled control.

To me the key part here is "labeled" - so I would rename this to `LabeledController`.
A link to this part of the standard in the JSDoc annotation would be good to have.

Reply:
Good point, how about `LabeledInputController` since the controller takes an input element in the constructor?

Reply:
Sounds great! This is quite self-descriptive indeed.

Reply:
Done.